apps/admin-web/src/screens/Events/EventList.tsx

```diff
@@ -49,4 +77,198 @@ const EventList: React.SFC<{}> = (): JSX.Element => {
+  if (loading) {
+    return (
+      <PageWrapper>
+        <h3>Loading...</h3>
+      </PageWrapper>
+    );
+  } else if (error) {
+    return (
+      <PageWrapper>
+        <h3>{error.toString()}</h3>
+      </PageWrapper>
+    );
+  } else {
+    const events: IEvent[] = data.events;
+    events.sort(
+      (a: any, b: any) =>
+        new Date(a.dateHosted).getTime() - new Date(b.dateHosted).getTime() //sorts by upcoming - later
+    );
+
+    const thisWeekEventFilter = events.filter(
+      (a: any) =>
+        new Date(a.dateHosted.split("T")[0]).getTime() -
+          new Date(a.dateHosted.split("T")[0]).getDay() * 86400000 ===
+        new Date(todaysISO.split("T")[0]).getTime() -
+          new Date(todaysISO.split("T")[0]).getDay() * 86400000 //Finds the monday of our week and the monday of the the events week and tests if its equal
+    );
+    const passedEventFilter = thisWeekEventFilter.filter(
+      (c: any) =>
+        new Date(c.dateHosted).getTime() -
+          new Date(todaysISO.split("T")[0]).getTime() > //Filters out dates that have passed
+        0
+    );
+
+    const listElements: JSX.Element[] = passedEventFilter.map(
+      (event: IEvent, index: number) => (
+        <Event event={event} index={index} key={event.id} />
+      )
+    );
+
+    return (
+      <PageWrapper>
+        <List>{listElements}</List>
+      </PageWrapper>
+    );
+  }
+};
+const NextWeekList: React.SFC<{}> = (): JSX.Element => {
+  const { loading, error, data }: any = useEventsQuery();
+
+  if (loading) {
+    return (
+      <PageWrapper>
+        <h3>Loading...</h3>
+      </PageWrapper>
+    );
+  } else if (error) {
+    return (
+      <PageWrapper>
+        <h3>{error.toString()}</h3>
+      </PageWrapper>
+    );
+  } else {
+    const events: IEvent[] = data.events;
+    events.sort(
+      (a: any, b: any) =>
+        new Date(a.dateHosted).getTime() - new Date(b.dateHosted).getTime() //sorts by upcoming - later
+    );
+
+    const nextWeekEventFilter = events.filter(
+      (a: any) =>
+        new Date(a.dateHosted.split("T")[0]).getTime() -
+          new Date(a.dateHosted.split("T")[0]).getDay() * 86400000 ===
+        new Date(todaysISO.split("T")[0]).getTime() +
+          7 * 86400000 -
+          new Date(todaysISO.split("T")[0]).getDay() * 86400000 //Finds our monday and event monday but 7 days in the future
+    );
+    const passedEventFilter = nextWeekEventFilter.filter(
+      (c: any) =>
+        new Date(c.dateHosted).getTime() -
+          new Date(todaysISO.split("T")[0]).getTime() >
+        0
+    );
+
+    const listElements: JSX.Element[] = passedEventFilter.map(
+      (event: IEvent, index: number) => (
+        <Event event={event} index={index} key={event.id} />
+      )
+    );
+
+    return (
+      <PageWrapper>
+        <List>{listElements}</List>
+      </PageWrapper>
+    );
+  }
+};
+
+const Future: React.SFC<{}> = (): JSX.Element => {
+  const { loading, error, data }: any = useEventsQuery();
```

Comment:
Instead of doing it this way, why not useevents query one time in the eventlist and pass the events for each in as props